### ISSUE TITLE
Dijkstra shortest path algorithm

### ISSUE THREAD

**Issue by Kanhakhatri065** [ephremdeme/data-structure-and-algorithms#237]
Implement Dijkstra shortest path algorithm in any language.

**Comment by amarks444** [comment]
I would love to do it in C++ if you want?

**Comment by kavya1229** [comment]
I will do it in python, please assign this to me

**Comment by Kanhakhatri065** [comment]
@amarks444 do it in C++ and @kavya1229 do it in python.

**Comment by amarks444** [comment]
[added Dijkstra in C++](https://github.com/amarks444/Data-Structure-Algorithms/blob/master/DIJKSTRA'SALGORITHM.CPP)

**Comment by Kanhakhatri065** [comment]
This looks good but can you also implement it for explicit graphs?

**Comment by amarks444** [comment]
Yes, I would love to do it, Can you provide some test cases for that..

**Comment by Kanhakhatri065** [comment]
g.addEdge(0, 1, 4); 

    g.addEdge(0, 7, 8); 

    g.addEdge(1, 2, 8); 

    g.addEdge(1, 7, 11); 

    g.addEdge(2, 3, 7); 

    g.addEdge(2, 8, 2); 

    g.addEdge(2, 5, 4); 

    g.addEdge(3, 4, 9); 

    g.addEdge(3, 5, 14); 

    g.addEdge(4, 5, 10); 

    g.addEdge(5, 6, 2); 

    g.addEdge(6, 7, 1); 

    g.addEdge(6, 8, 6); 

    g.addEdge(7, 8, 7); 

The first one is source node, the second one is destination node and third value is weight of that edge.

**Comment by rishinpandit09** [comment]
hey i will do this in python can u pls add me

**Comment by amarks444** [comment]
I have Updated Dijkstras for Implicit Graphs using Set in STL ..**#305** 
You can Merge it..

**Comment by amarks444** [comment]
> I have Updated Dijkstras for Implicit Graphs using Set in STL ..**#305**
> You can Merge it..

 You can Merge it..

**Comment by Kanhakhatri065** [comment]
@amarks444 done.
